packages/react-native-client/ios/RNFishjamClient.swift

```diff
@@ -402,11 +394,11 @@ class RNFishjamClient: FishjamClientListener {
     func getPeers() throws -> [[String: Any?]] {
         let endpoints = RNFishjamClient.getLocalAndRemoteEndpoints()
         return endpoints.compactMap { endpoint in
-            [
+            return [
```

Comment:
**question:** 
Is return statement needed here?

Reply:
I forgot to remove that after testing values in this map. Thanks!